.vscode/settings.json

```diff
@@ -2,6 +2,7 @@
   "cSpell.words": [
     "backoff",
     "casted",
-    "mappish"
+    "mappish",
+    "Planable"
```

Comment:
I'm pretty sure this isn't a word 😆

Reply:
So is my spelling checker, which is why I added it to the list :)